scripts/DeployMultisigIsm.s.sol

```diff
@@ -4,29 +4,42 @@ pragma solidity ^0.8.17;
 import "../lib/forge-std/src/Script.sol";
 import "../lib/forge-std/src/console.sol";
 
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
 import {ConfigLib} from "../lib/ConfigLib.sol";
 import {CheckLib} from "../lib/CheckLib.sol";
 import {DeployLib} from "../lib/DeployLib.sol";
 import {MultisigIsm} from "@hyperlane-xyz/core/contracts/isms/MultisigIsm.sol";
 import {TestRecipient} from "@hyperlane-xyz/core/contracts/test/TestRecipient.sol";
 
-// TODO: Deploy test recipient, maybe write to networks.
 contract DeployMultisigIsm is Script {
-    using CheckLib for ConfigLib.MultisigIsmConfig;
-    using DeployLib for ConfigLib.MultisigIsmConfig;
+    using Strings for uint256;
 
-    function run() public {
-        address owner = vm.envAddress("OWNER");
+    using DeployLib for ConfigLib.Multisig;
+    using ConfigLib for ConfigLib.Multisig;
+    using CheckLib for MultisigIsm;
+
+    function run() public returns (MultisigIsm ism) {
         string[] memory remotes = vm.envString("REMOTES", ",");
-        ConfigLib.MultisigIsmConfig memory config = ConfigLib
-            .readMultisigIsmConfig(vm, remotes);
+        address owner = vm.envAddress("OWNER");
+
+        ConfigLib.Multisig memory config = ConfigLib.readMultisigIsmConfig(
+            vm,
+            remotes,
+            owner
+        );
 
-        vm.startBroadcast();
+        string memory key = uint256(config.digest()).toString();
```

Comment:
nit: Does this mean we can't deploy two MultisigIsms with the same config to two different chains?